validator/server_api/json.go

```diff
@@ -52,16 +50,13 @@ func ValidationInputToJson(entry *validator.ValidationInput) *ValidationInputJso
 		encData := base64.StdEncoding.EncodeToString(binfo.Data)
 		res.BatchInfo = append(res.BatchInfo, BatchInfoJson{binfo.Number, encData})
 	}
-	for call, wasm := range entry.UserWasms {
-		callBytes := arbmath.Uint16ToBytes(call.Version)
-		callBytes = append(callBytes, call.CodeHash.Bytes()...)
-		encCall := base64.StdEncoding.EncodeToString(callBytes)
+	for moduleHash, info := range entry.UserWasms {
+		encModuleHash := base64.StdEncoding.EncodeToString(moduleHash[:])
```

Comment:
I think typically we use hex for hash keys in this JSON. I'd stick with that (just using a common.Hash as the key should work).

Reply:
Ah that's nice, updated!